inspector/ios-inspector/ForgeModule/tabs_WKWebViewController.m

```diff
@@ -327,9 +282,13 @@ - (void) createNavigationBarVisualEffect:(UIView*)theWebView {
     _blurView = [[UIView alloc] init];
     _blurView.userInteractionEnabled = NO;
     _blurView.backgroundColor = [UIColor clearColor];
-    _blurViewVisualEffect = [[UIVisualEffectView alloc] initWithEffect:[UIBlurEffect effectWithStyle:UIBlurEffectStyleRegular]];
-    _blurViewVisualEffect.userInteractionEnabled = NO;
-    [_blurView addSubview:_blurViewVisualEffect];
+    
+    if (self.navigationBarIsOpaque == NO) {
+        _blurViewVisualEffect = [[UIVisualEffectView alloc] initWithEffect:[UIBlurEffect effectWithStyle:UIBlurEffectStyleRegular]];
```

Comment:
@antoinevg I reverted the rename. My intention was that the _blurViewVisualEffect is only applied in case the navigationbar is not opaque. Otherwise it's just tinted.